src/main/kotlin/bandcampcollectiondownloader/BandcampAPIConnector.kt

```diff
@@ -99,12 +100,12 @@ class BandcampAPIConnector constructor(private val bandcampUser: String, private
                                         .ignoreContentType(true)
                                         .timeout(timeout)
                                         .cookies(cookies)
+                                        .method(Method.POST)
                                         .requestBody("{\"fan_id\": $fanId, \"older_than_token\": \"$lastToken\"}")
-                                        .post()
+                                        .execute()
```

Comment:
Sure why not but what does this change exactly? How is `method`+`execute` different from `post`?

Reply:
So `post()` executes the request _and_ parses the result while `execute()` just executes the request. With my use case, jsoup was not parsing the result correctly. 

This is what `theRest!!.wholeText()` was returning before `gson.fromJson` executed.
```json
{"file":{"mp3-v0":"https://bandcamp.com/stream_redirect?enc=mp3-v0&track_id=29128347&ts=1600291760&t=c2360d89add21e40f7fe234fab22ac61bf9cda63"},"title":"増>>>>減<<<
```

Causing this error `com.google.gson.stream.MalformedJsonException: Unterminated string at line 1 column 95318 path $.tracklists.a2506905463[17].title`